plugins/modules/ndo_physical_interface.py

```diff
@@ -0,0 +1,411 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Anvitha Jain (@anvjain) <anvjain@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: ndo_physical_interface
+short_description: Manage Physical Interfaces on Cisco Nexus Dashboard Orchestrator (NDO).
+description:
+- Manage Physical Interfaces on Cisco Nexus Dashboard Orchestrator (NDO).
+- This module is only supported on ND v3.1 (NDO v4.3) and later.
+author:
+- Anvitha Jain (@anvjain)
+options:
+  template:
+    description:
+    - The name of the template.
+    - The template must be a fabric resource policy template.
+    type: str
+    required: true
+  physical_interface:
+    description:
+    - The name of the Physical Interface.
+    type: str
+    aliases: [ name ]
+  physical_interface_uuid:
+    description:
+    - The UUID of the Physical Interface.
+    - This parameter is required when the O(physical_interface) needs to be updated.
+    type: str
+    aliases: [ uuid ]
+  description:
+    description:
+    - The description of the Physical Interface.
+    type: str
+  nodes:
+    description:
+    - The node IDs where the Physical Interface policy will be deployed.
+    type: list
+    elements: int
+  interfaces:
+    description:
+    - The interface names where the policy will be deployed.
+    - The old O(interfaces) will be replaced with the new O(interfaces) during an update.
+    type: list
+    elements: str
+  physical_interface_type:
+    description:
+    - The type of the interface policy group.
+    type: str
+    choices: [ physical, breakout ]
+  physical_policy_uuid:
+    description:
+    - The UUID of the Interface Setting Policy.
+    - This is only required when creating a new Port Channel Interface.
+    - This parameter is required when O(physical_interface_type) is C(physical).
+    - This parameter can be used instead of O(physical_policy).
```

Comment:
- This parameter can be used instead of O(physical_policy).
The above is not clear

Reply:
physical_policy_uuid can be used instead of physical_policy
and
physical_policy can be used instead of physical_policy_uuid

Reply:
Is the below statement correct?
- This is only required when creating a new Interface Setting Policy.

Reply:
Discussed online.